### DISCUSSION TITLE
Run segments of code enclosed between same magnitude markdown cells

### DISCUSSION THREAD

**Discussion by czephyr** [microsoft/vscode-jupyter#10463] [New Ideas]
We already have the possibility of visually folding code that is enclosed between title markdown cells. 

Would it be possible to have a run button to run all of the cells linked to a certain markdown title?

**Comment by rchiodo** [comment]
You might try the powertoys extension:
https://marketplace.visualstudio.com/items?itemName=ms-toolsai.vscode-jupyter-powertoys

It has a way to mark cells as part of a group. You can then execute all the cells in a group.
